config/src/wasm.rs

```diff
@@ -3,7 +3,7 @@
 use iroha_config_base::derive::{Documented, Proxy};
 use serde::{Deserialize, Serialize};
 
-const DEFAULT_FUEL_LIMIT: u64 = 1_000_000;
+const DEFAULT_FUEL_LIMIT: u64 = 15_000_000;
```

Comment:
really?

Reply:
Yes, but actually no, haha. I changed `cli/samples.rs` too. It was using `10_000_000` as a fuel limit and that was used for test.

I have spend some time trying to figure out where is this limitation being set. Then I've found this two unrelated places and decided to remove the code from `cli/samples.rs`.

So in fact the increase is not **15 times** but in fact **1.5 times**

Reply:
For now these numbers are arbitrary and don't matter. In the future each of these constants must be documented such that each increase is informed both for the reviewer and the creator.